src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -1737,5 +1785,48 @@ enum ReplicationMode
         /// </summary>
         Logical
     }
+
+    /// <summary>
+    /// Specifies server type preference.
+    /// </summary>
+    public enum TargetSessionAttributes : byte
```

Comment:
@vonzshik there is still the question of whether this should be an enum or a string (in case PG evolves it later). My gut feeling is that to be on the safe side, it's probably better to make this a string - just like the libpq one. Thoughts?